### ISSUE TITLE
StoreKit -> SKStoreReviewController  -> requestReviewInScene missing

### ISSUE THREAD

**Issue by extrawurst** [madsmtm/objc2#541]
see https://developer.apple.com/documentation/storekit/skstorereviewcontroller/3566727-requestreviewinscene?language=objc

as of `0.0.4` we seem to only provide deprecated [requestReview](https://developer.apple.com/documentation/storekit/skstorereviewcontroller/2851536-requestreview?language=objc) and `requestReviewInScene` is missing

**Comment by madsmtm** [comment]
Yeah, this is iOS only, and the APIs are currently generated assuming macOS

**Comment by extrawurst** [comment]
oh i see. so only 100% identical Apis will work on both? like the discussion we had on the notifications?

**Comment by madsmtm** [comment]
Yup, exactly.

**Comment by madsmtm** [comment]
Just to be clear, in the interim you can do:

```rust
/// # Safety
///
/// Must be run on the main thread, and must be passed a `UIWindowScene`.
unsafe fn request_review_in_scene(window_scene: &NSObject) {
    unsafe { msg_send![SKStoreReviewController::class(), requestReviewInScene: window_scene] }
}
```

**Comment by extrawurst** [comment]
> in the interim you can do

that's great! thanks for the clarification

**Comment by madsmtm** [comment]
I'll close this as tracked under https://github.com/madsmtm/objc2/issues/408 (since what we need to do to fix this is run the header translator for each platform, and then merge the result).
